gramex/sm_api.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+from statsmodels import api as sm
```

Comment:
Statsmodels should be optional. Gramex should run without statsmodels as a dependency.